templates/prosody.service.j2

```diff
@@ -0,0 +1,78 @@
+# this file is from /usr/local/share/prosody-modules/misc/systemd/
+
+[Unit]
+### see man systemd.unit
+Description=Prosody XMPP Server
+Documentation=https://prosody.im/doc
+
+[Service]
+### See man systemd.service ###
+# With this configuration, systemd takes care of daemonization
+# so Prosody should be configured with daemonize = false
+Type=simple
+
+# Not sure if this is needed for 'simple'
+PIDFile=/var/run/prosody/prosody.pid
+
+# Start by executing the main executable
+ExecStart=/usr/bin/prosody
+
+ExecReload=/bin/kill -HUP $MAINPID
+
+# Restart on crashes
+Restart=on-abnormal
+
+# Set O_NONBLOCK flag on sockets passed via socket activation
+NonBlocking=true
+
+### See man systemd.exec ###
+
+WorkingDirectory=/var/lib/prosody
+
+User=prosody
+Group=prosody
+
+UMask=0027
+
+# Nice=0
+
+# Set stdin to /dev/null since Prosody does not need it
+StandardInput=null
+
+# Direct stdout/-err to journald for use with log = "*stdout"
+StandardOutput=journal
+StandardError=inherit
+
+# This usually defaults to 4000 or so
+# LimitNOFILE=20000
```

Comment:
As stated above, remove this comment and if clause and just give a meaningful default value.